<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Simplify ComboBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnIniSImplify">
        <property name="text">
         <string>Ini</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnCleanSImplify">
        <property name="text">
         <string>Clean</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="chkBoxEditable">
        <property name="text">
         <string>Editable</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QComboBox" name="comboSimplify"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>GroupBox With User's Data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnIniCities">
        <property name="text">
         <string>Ini Cities + Code</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboCities"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>QPlainTextEdit</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnAddText">
        <property name="text">
         <string>Add Text To ComboBox</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="chkBoxReadOnly">
        <property name="text">
         <string>Read Only</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnCleanText">
        <property name="text">
         <string>Clean Text</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCleanSImplify</sender>
   <signal>clicked()</signal>
   <receiver>comboSimplify</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
